### ISSUE TITLE
[FEATURE] `<slot></slot>` should not trigger empty tags warning

### ISSUE THREAD

**Issue by mangelozzi** [djlint/djLint#866]
The native `slot` component is for slotting content, and this is probably the most common usage of it:
```
<slot></slot>
```

It raises `empty pair found consider removing [H020]`, which I think should not be displayed if it is a `slot` element.

**Comment by mangelozzi** [comment]
Thank you 👍
